suites/nautilus/rgw/tier-1_rgw_ssl_containerized-regression.yaml

```diff
@@ -0,0 +1,106 @@
+#===============================================================================================
+# Tier-level: 1
+# Test-Suite: tier-1_rgw_ssl_containerized-regression.yaml
+# Test-Case: adding IO operations with SSL frontend
+#
+# Cluster Configuration:
+#    cephci/conf/nautilus/rgw/sanity-rgw.yaml
+#
+#    11-Node cluster(RHEL-7.9 and above)
+#    3 MONS, 1 MDS, 1 MGR, 3 OSD and 1 RGW service daemon(s)
+#
+# Test Steps:
+#   (1) Install Pre-requisites, and Deploy Ceph using ceph-ansible with CDN/GA container image
+#   (2) check cluster health.
+#   (3) perform RGW IO operations with SSL frontend
+#===============================================================================================
+tests:
+   - test:
+       name: install ceph pre-requisites
+       module: install_prereq.py
+       abort-on-fail: true
+   - test:
+      name: containerized ceph ansible
+      polarion-id: CEPH-83571503
+      module: test_ansible.py
+      config:
+        ansi_config:
+            ceph_test: True
+            ceph_origin: distro
+            ceph_stable_release: nautilus
+            ceph_repository: rhcs
+            osd_scenario: lvm
+            osd_auto_discovery: False
+            journal_size: 1024
+            ceph_stable: True
+            ceph_stable_rh_storage: True
+            ceph_docker_image: rhceph/rhceph-3-rhel7
+            ceph_docker_image_tag: latest
+            containerized_deployment: true
+            ceph_docker_registry: registry.access.redhat.com
```

Comment:
Sorry, I may have missed this earlier. This not being an upgrade scenario, I believe we don't need these configurations/parameters. Also, 11 nodes is very high on resources. Could we revisit the minimum resource requirements?